src/smoosh/src/smoosh_channel.erl

```diff
@@ -177,21 +187,34 @@ handle_info(check_window, State0) ->
     erlang:send_after(60 * 1000, self(), check_window),
     {noreply, FinalState};
 
+handle_info(persist_queue, State0) ->
+    #state{waiting=Queue} = State0,
+    smoosh_priority_queue:write_to_file(Queue),
+    AutoSave = config:get_integer("smoosh", "save_time", ?DEFAULT_AUTOSAVE_PERIOD),
```

Comment:
Naming is hard!!!

`save_time` doesn't reflect what the setting is doing. I am thinking something like the following would be better:

- `state_checkpoint_interval_in_sec`
- `state_autosave_interval_in_sec`